src/schemas/components/SchemaFormPropertyAnyOf.vue

```diff
@@ -1,7 +1,11 @@
 <template>
   <keep-alive>
-    <SchemaFormProperty :key="selectedPropertyIndexValue" v-model:value="value" v-bind="{ property, required, errors }" class="schema-form-property-any-of-input">
-      <p-button-group v-model="selectedPropertyIndex" :options="options" small class="mb-2" />
+    <SchemaFormProperty :key="selectedPropertyIndexValue" :value="internalValue" v-bind="{ property, required, errors }" class="schema-form-property-any-of-input" @update:value="updateValue">
+      <template #default="{ kind }">
+        <template v-if="kind === 'none'">
+          <p-button-group v-model="selectedPropertyIndex" :options="options" small class="mb-2" />
+        </template>
+      </template>
```

Comment:
This also removes the button group when the form isn't being used. Which was a confusing piece of ux if you were using json or another kind of input.